src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+function ifSuccess(successMessage) {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = successMessage;
+  document.body.appendChild(div);
+}
+
+function ifError(errorMessage) {
+  const div = document.createElement('div');
+
+  div.classList.add('message', 'error-message');
+  div.textContent = errorMessage;
+  document.body.appendChild(div);
+}
```

Comment:
The ifError function is very similar to the ifSuccess function. Consider combining these two functions into one to reduce code duplication. You could pass in the message and the class names as parameters.